libs/sdk-backend-tiger/api/sdk-backend-tiger.api.md

```diff
@@ -81,32 +93,27 @@ export interface IDataSource {
 // @internal (undocumented)
 export interface IDataSourceApiResult {
     // (undocumented)
-    data?: IDataSourceConnectionInfo | IDataSourceTestConnectionResponse;
+    data?: IDataSourceConnectionInfo;
     // (undocumented)
     errorMessage?: string;
 }
 
 // @internal (undocumented)
 export interface IDataSourceConnectionInfo {
     // (undocumented)
-    data: {
-        attributes: {
-            name: string;
-            schema: string;
-            type: IDataSourceType;
-            url?: string;
-            username?: string;
-        };
-        id: string;
-        meta?: {
-            permissions?: IDataSourcePermission[];
-        };
-        type: JsonApiDataSourceOutTypeEnum;
-    };
-    // (undocumented)
-    links?: {
-        self: string;
-    };
+    id: string;
```

Comment:
I'm afraid of these changes, there are apps which are relying and reusing these types heavily (see https://github.com/gooddata/gdc-msf-modeler/pull/1163/files which is under review now for example) and these would have to be reworked a lot.
Are you preparing updates to those parent apps as well? (on top of the mentioned 1163 pr for example in modeler)